src/components/Login/EnterPhonePanel.tsx

```diff
@@ -0,0 +1,69 @@
+import { useState } from 'react'
+
+import Form from '@/components/Login/LoginForm'
+import Button from '@/components/UI/button'
+import { SelectOption } from '@/types/types'
+
+export interface EnterPhonePanelInterface {
+  onClick: (phoneNumber: string) => void
+}
+
+const EnterPhonePanel = ({ onClick }: EnterPhonePanelInterface) => {
+  const [displaynumber, setdisplaynumber] = useState('')
+  const [countrycode, setcountrycode] = useState('')
+
+  const countryCodeSelectOptions: SelectOption[] = [
+    { value: '+61', label: 'Australia (+61)' }
+  ]
+
+  function handleSubmit(event: React.ChangeEvent<HTMLFormElement>) {
+    onClick(countrycode + displaynumber)
+    event.preventDefault()
+    event.target.reset()
+  }
+
+  function formFilled() {
+    return displaynumber !== ''
+  }
+
+  return (
+    <div className='flex w-1/4 flex-auto flex-col'>
+      <form
+        onSubmit={handleSubmit}
+        className='m-5 flex h-full flex-auto flex-col items-center justify-around'
+      >
+        <div className='flex justify-center'>
+          <Form
```

Comment:
Please remove the country code field, a single input field will be enough (with phone number validation)

Reply:
@scratchclaggy I think I will need the phone number with the country code when passing it to the phone auth function. Might be better to leave it there to let the users know?

Reply:
Ah okay fair enough then. Does the validation library I mentioned (or some other one) come with an array of country codes? I know that we're only targeting Australian users, but it is possible for someone living in Perth to have an international number, so I'm a little nervous about only offering +614.